### DISCUSSION TITLE
ha addons install

### DISCUSSION THREAD

**Discussion by LeroyLam** [Bepacom-Raalte/bepacom-HA-Addons#76] [Q&A]
middag,

ik ben een vraag ben nu al een week bezig.
maar het lukt me met geen mogelijkheid om het werkend te krijgen op mijn home a.
hij blijf steeds hangen bij dat die het netwerk niet kan vinden bij addon bepacom BACnet/ip interface.
[IMG-20240930-WA0002[1].pdf](https://github.com/user-attachments/files/17191098/IMG-20240930-WA0002.1.pdf)
wat doe ik fout ?

Bedankt alvast

**Comment by GravySeal** [comment]
Goedemiddag Leroy,

Heb je de addon toevallig al geïnstalleerd? Zo ja, zou je daarvan de logs eens kunnen delen?

**Reply by LeroyLam** [reply]
het is al gelukt maar nu heb ik die ede file geladen en krijg wat componenten te zien alleen krijg ik geen waardes door zie wel dat ik bacnet mijn klepje zie en opnemers maar zie geen waardes

**Reply by GravySeal** [reply]
Hallo Leroy,

De EDE bestanden zijn bedoeld als placeholder voor het echte apparaat (voor bijvoorbeeld het genereren van entities).
Als je live data wilt zien, moet het apparaat gedetecteerd worden op het netwerk.
Als je het EDE bestand uit de add-on verwijderd, en je eens op who-is drukt, verschijnt het echte apparaat dan in de add-on?

**Reply by LeroyLam** [reply]
nee dan gebeurt er niks? 
![image](https://github.com/user-attachments/assets/3a8eaa65-5102-4dc4-b551-662fdfff71f1)
![image](https://github.com/user-attachments/assets/5b7329ae-d2fe-4a5b-984e-269d4c30b14e)

de eco paneel zie die wel 

![image](https://github.com/user-attachments/assets/eaff4028-47a9-4520-ab71-9b8e4ac40525)

![image](https://github.com/user-attachments/assets/eea7ced0-0ed9-4fa3-85b9-bc54f7383e8c)

![image](https://github.com/user-attachments/assets/6173e9c5-f63e-41e6-a175-4f22054cf614)


hier zie ik ook niks bij komen staan

**Reply by LeroyLam** [reply]
> j

ja dan zie ik ze wel komen moet ik ze nu allemaal een naam gaan geven of is dat makkelijk te doen ?

<img width="660" height="726" alt="image" src="https://github.com/user-attachments/assets/372a6605-f48f-4ca6-bd33-e40b7f3f6e4a" />

**Comment by GravySeal** [comment]
Welk IP adres heeft je BACnet apparaat?

**Reply by LeroyLam** [reply]
192.168.14.228

**Reply by GravySeal** [reply]
Dat is het ip dat je bij de add-on hebt ingevoerd.
Zorg dat de add-on hetzelfde ip adres ingevuld krijgt als je Home Assistant apparaat.
Het BACnet apparaat dat je uit wil lezen zal een ander ip adres moeten hebben.

**Reply by LeroyLam** [reply]
ik heb nu dit 
![image](https://github.com/user-attachments/assets/3c7e26f9-d273-4fd9-967f-4fcd82852871)
![image](https://github.com/user-attachments/assets/14f47c07-5bfd-470d-afca-15d5e60f4bef)
![image](https://github.com/user-attachments/assets/d4eb6245-5d41-4a23-ac62-20c701deddb0)

maar als ik dan op zoek ga naar mijn regeling dan zeg die dit 
![image](https://github.com/user-attachments/assets/9325a48e-9260-4b94-9f5b-2ca5a4f9591c)
en het ip is dan 192.168.14.228 die is van de regeling.
weet niet wat er mis ga?

**Reply by GravySeal** [reply]
Oke, dat is prima, zou je van het ip adres 192.168.14.95/24 kunnen maken?

De integration is nog niet zo belangrijk. Via de web UI moeten we eerst je BACnet apparaat kunnen zien.

**Reply by LeroyLam** [reply]
dan geeft die het zelf weer dat die niet kan inloggen heel gek ?

**Reply by GravySeal** [reply]
In de add-on, zie jij je apparaat verschijnen zoals dit?
![image](https://github.com/user-attachments/assets/39999606-6236-48c6-aba8-c6876c07719b)

**Reply by LeroyLam** [reply]
nee niks
![image](https://github.com/user-attachments/assets/72c20633-689d-4896-a7f6-8f882ffb587c)

**Reply by GravySeal** [reply]
Heb je ooit eens van YABE gehoord? Wat als je daarmee je netwerk scant?

**Reply by LeroyLam** [reply]
Dank je ga er morgen is mee kijken of ik er wat mee kan vinden op het netwerk bedankt voor de vriendenlijk
Ondersteuningen weer

**Reply by GravySeal** [reply]
Top, geen probleem! Hopelijk kunnen we het aan de praat krijgen

**Reply by LeroyLam** [reply]
heb net gekken met InneaBACnetExplorer maar dan vind ik de regelaar wel ?
en kan ik hem uitlezen met wat er allemaal op zit.
wat gaat er dan fout zie daar wel de temp van me ketel en van de vloerverwarming.

**Reply by GravySeal** [reply]
Zou het loggingsniveau van de add-on eens op debug kunnen zetten en dan via de web UI de logs downloaden?
Graag deze eens delen.
Als ik het dus goed begrijp zit je Home Assistant op 192.168.14.95 en je regelaar op 192.168.14.228.
Is je Home Assistant installatie een eigen apparaat of draai je het via een virtual machine?
Staan op beide apparaten de default gateway hetzelfde, en werken ze met hetzelfde subnet (255.255.255.0)?

**Reply by LeroyLam** [reply]
hoe heb jij hem draaien dan als ik vragen mag ?

**Reply by GravySeal** [reply]
Ik heb een versie draaien aan de zaak onder een router met 2 regelaars eronder. Thuis gebruik ik hem als testversie met een simulator applicatie om het een en ander te kunnen testen.

Wanneer je YABE installeert, krijg je Bacnet.Room.Simulator ook mee. Deze zou je eens kunnen proberen te draaien op je computer, daarna eens het netwerk scannen en kijken wat je tegenkomt.

Ik zie in de andere reply dat je Raspberry Pi een IP heeft met 229? Dit is dan je Home Assistant? 
Heb je ook al eens geprobeerd om "auto" in te vullen bij het IP adres in de add-on?

**Reply by LeroyLam** [reply]
ik heb het vermoeden dat het niet werk op een raspberry pi ga het vanavond op een vm proberen kijken wat ik dan tegen kom of op een raspberry pi3 maar weer bedankt voor de hulpsaam zijn op deze pagina.

**Reply by GravySeal** [reply]
Bij mij werkt hij op een VM ook, zolang de ethernet adapter maar bridged draait.
Als je dezelfde PC gebruikt als waarbij je wel je regelaar zag, zou de add-on in de VM hem ook moeten kunnen zien.

![image](https://github.com/user-attachments/assets/3987cee4-e277-492f-81cb-322ff77b2fef)

Ik heb de add-on ook getest met een raspberry pi 3, dus zou het raar vinden als het aan een raspberry pi 4 zou liggen.
Wanneer je de add-on opstart en je bij de web UI kan komen, zou je eens een screenshot kunnen maken van de add-on logs?

**Reply by LeroyLam** [reply]
heb hem op de vm gelijk werkend.
ik denk dat het licht aan de 4 dus ga van het weekend het even op de 3 testen 
kijken of het daar wel mee werkt heb niks anders gedaan als op de 4 en op die vm werkten hij gelijk
ik hou je op de hoogte nogmaals bedankt voor je tijd.

**Reply by GravySeal** [reply]
Oke, top! Ben wel benieuwd waarom de add-on niet werkte, dus laat maar weten.

**Reply by LeroyLam** [reply]
nou heel het weekend bezig geweest op een pi3 werkt die gelijk,
maar op de pi 4 niet.
heb zelf een back up werkend over gezet op de 4 zelf dan komt der niks binnen.
heb ook geen idee of het aan de pi4 licht of dat het de software is ?

**Reply by GravySeal** [reply]
Ik zou eens een raspberry pi 4 of 5 aan moeten schaffen om te kijken of het daar aan ligt. Zou het wel raar vinden als alleen een raspberry pi 4 er last van heeft, maar het kan. In ieder geval bedankt voor het testen!

**Reply by LeroyLam** [reply]
waarom werken de ede files niet zit er een fout in of doe ik wat fout 
<img width="1825" height="681" alt="image" src="https://github.com/user-attachments/assets/85ef9fdd-0157-4623-9b0f-00074816947d" />

ze staan goed maar heb geen idee wat ik fout doe

**Comment by LeroyLam** [comment]
hij draait op een raspberry pi 4 klopt heb alleen de ip adres van de raspberry pi4 dicht bij gezet naar 229.instellignen kloppen gateway en het subnet zijn beiden het zelfden 




[bacnet_addon_logs (2).txt](https://github.com/user-attachments/files/17517227/bacnet_addon_logs.2.txt)
